### DISCUSSION TITLE
[ 资讯 ] 治安局将开展宣传活动

### DISCUSSION THREAD

**Discussion by share121** [share121/inter-knot#15] [其他]
![image](https://github.com/user-attachments/assets/06cb5c4b-5b30-4cc0-ae7d-75755bb85e26)

近日，雅努斯区治安局发出公告，将在全区范围内开展治安巡防宣传月活动。
据悉，本次活动由雅努斯区总局贾斯汀·布林格副总长官直接发起，旨在提升居民安全防范意识，促进治安局与辖区居民的交流，有效打击犯罪。

值得注意的是，在公告中列出的被打击对象中，绳网也榜上有名。

**Comment by share121** [comment]
明白了，绳匠生意又要变得难做了。

**Comment by share121** [comment]
明白了，市政换届选举快来了。

**Comment by share121** [comment]
是我的错觉吗，最近好像总是在各种地方看见布林格的脸。

**Comment by share121** [comment]
你不会不知道吧，布林格可是竞选雅努斯区治安局总长官的热门人选。

**Comment by share121** [comment]
最近那么多邦布盗窃案是该管一管了！

**Reply by YumeRin39Hana** [reply]
我就是绳匠！你们这群没良心的治安官就知道抓我们拿业绩！一点公德心都没有，告诉你们，我里边有人，就算抓到我你们也只能看着我平安无事出来的，哈哈哈哈哈~

**Comment by YumeRin39Hana** [comment]
我就是绳匠！你们这群没良心的治安官就知道抓我们拿业绩！一点公德心都没有，告诉你们，我里边有人，就算抓到我你们也只能看着我平安无事出来的，哈哈哈哈哈~ @雅努斯区治安局

**Comment by DeepslateQAQ** [comment]
Fairy：已将该信息标记为垃圾信息。
